resolwe_bio/processes/slamdunk/alleyoop_summary.py

```diff
@@ -0,0 +1,64 @@
+"""Run Alleyoop summary tool on Slamdunk results."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, Process, StringField
+
+
+class AlleyoopSummary(Process):
+    """Run Alleyoop summary."""
+
+    slug = 'alleyoop-summary'
+    process_type = 'data:alleyoop'
+    name = 'Alleyoop summary'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/slamdunk:1.0.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    entity = {
+        'type': 'sample',
+    }
+    data_name = '{{ reads|sample_name|default("?") }}'
+    version = '1.0.0'
+
+    class Input:
+        """Input fields for AlleyoopSummary."""
+
+        slamdunk = DataField('alignment:bam:slamdunk', label='Slamdunk results')
+
+    class Output:
+        """Output fields to process AlleyoopSummary."""
+
+        table = FileField(label='Tab-separated file with mapping statistics')
+        species = StringField(label="Species")
+        build = StringField(label="Build")
+
+    def run(self, inputs, outputs):
+        """Run analysis."""
+        basename = os.path.basename(inputs.slamdunk.bam.path)
+        assert basename.endswith('.bam')
+        name = basename[:-4]
+
+        (Cmd['ln']['-s', inputs.slamdunk.tcount.path, f'{name}_tcount.tsv'])()
+
+        args = [
+            '-o', f'{name}_summary.txt',
```

Comment:
`f'{name}_summary.txt'` is used twice (also in `outputs.table`). Consider putting this into a variable.